tests/pytest/in_person/test_views.py

```diff
@@ -49,6 +49,31 @@ def test_eligibility_logged_in(admin_client):
     assert response.template_name == "in_person/eligibility.html"
 
 
+@pytest.mark.django_db
+def test_index_filtering_fows(mocker, model_TransitAgency, admin_client):
```

Comment:
This one was fine / more correct as `test_eligibility_logged_in_filtering_flows` since it's testing the `eligibility` view function.

```suggestion
def test_eligibility_logged_in_filtering_flows(mocker, model_TransitAgency, admin_client):
```